### DISCUSSION TITLE
v10 Terrain - Camera position by altitude

### DISCUSSION THREAD

**Discussion by dpyeates** [rnmapbox/maps#2118] [Q&A]
I am using v10 terrain view and would like to position the camera at a latitude/longitude, heading and altitude.

I can see how I set all apart from altitude?

**Comment by mfazekas** [comment]
You cannot do that at the moment. I guess play with pitch and zoom to simulate the effect of looking down from a specific altitude - https://docs.mapbox.com/ios/maps/guides/camera-and-animation/camera/

Native side has [FreeCameraOptions](https://docs.mapbox.com/android/maps/api/10.7.0/mapbox-maps-android/com.mapbox.maps/-free-camera-options/index.html), but not all animations like `flyTo` supports those

**Reply by dpyeates** [reply]
Would it be possible to add the setLocation and setPitchBearing FreeCameraOptions to the package to make this available to RN?

**Reply by bradstiff** [reply]
I also very much need to set the camera position in terms of lat, lng and altitude.
